README.md

```diff
@@ -1,217 +1,256 @@
 # vrs-python
 
-[![PyPI version](https://badge.fury.io/py/ga4gh.vrs.svg)](https://pypi.org/project/ga4gh.vrs)
-
-vrs-python provides Python language support for the [GA4GH Variation
+**VRS-Python** provides Python language support for the [GA4GH Variation
```

Comment:
"Python language support" is a little vague. It's really the reference implementation, and happens to be in Python.

Maybe:

`...provides Python language support and a reference implementation for the...` ?